docker/playground/compose.yml

```diff
@@ -78,3 +78,15 @@ services:
       - metastore
       - minio
       - zookeeper
+
+  kyuubi-client:
+    image: nekyuubi/kyuubi-playground-kyuubi:${KYUUBI_VERSION}
+    container_name: kyuubi-client
+    hostname: kyuubi-client
+    volumes:
+      - ./image/load-dataset-tpcds-tiny.sql:/etc/kyuubi/conf/load-dataset-tpcds-tiny.sql
+      - ./image/load-dataset-tpch-tiny.sql:/etc/kyuubi/conf/load-dataset-tpch-tiny.sql
+      - ./image/load-dataset.sh:/opt/kyuubi/bin/load-dataset.sh
```

Comment:
we should not put the scripts to `/opt/kyuubi/bin`, it may confuse users. 
let's put them into `/opt/bin` instead